src/main/java/cc/hyperium/Hyperium.java

```diff
@@ -215,27 +241,22 @@ public void init(InitializationEvent event) {
             SplashProgress.setProgress(6, I18n.format("splashprogress.registeringlisteners"));
             minigameListener = new MinigameListener();
             EventBus.INSTANCE.register(minigameListener);
-            EventBus.INSTANCE.register(new ToggleSprintContainer());
             EventBus.INSTANCE.register(notification);
-
+            EventBus.INSTANCE.register(confirmation);
+            EventBus.INSTANCE.register(statTrack);
+            EventBus.INSTANCE.register(new ToggleSprintContainer());
             EventBus.INSTANCE.register(CompactChat.getInstance());
             EventBus.INSTANCE.register(CONFIG.register(FPSLimiter.getInstance()));
-            EventBus.INSTANCE.register(confirmation);
             EventBus.INSTANCE.register(new BlurHandler());
             EventBus.INSTANCE.register(new CommandUpdate());
             EventBus.INSTANCE.register(new ThankWatchdog());
-
-            // Register statistics tracking.
-            EventBus.INSTANCE.register(statTrack);
-            CONFIG.register(statTrack);
             CONFIG.register(new ToggleSprintContainer());
+            CONFIG.register(statTrack);
 
             SplashProgress.setProgress(7, I18n.format("splashprogress.startinghyperium"));
-            LOGGER.info("[Hyperium] Started!");
             Display.setTitle("Hyperium " + Metadata.getVersion());
```

Comment:
Why has this been removed?

Reply:
@asbyth I think he wanted this fixed also